### ISSUE TITLE
[Bug]: 0.4.7 causes errors in old project

### ISSUE THREAD

**Issue by altano** [jetpack-io/devbox#925]
**Current Behavior (bug)**
I had a project that was last used successfully with whatever the last version of devbox was (0.4.6?) but the 0.4.7 that just came out broke it.

Similar to https://github.com/jetpack-io/devbox/issues/829, when I enter the old project the direnv init causes an error:

```
➜  src cd docker
direnv: loading ~/src/docker/.envrc
direnv: using devbox
direnv: using nix .devbox/gen/shell.nix
/bin/bash: line 9: export: `.': not a valid identifier
/bin/bash: line 9: export: `/Users/alan/src/docker/.devbox/gen/scripts/.hooks.sh': not a valid identifier
```

That .hooks.sh file does exist, fwiw, but is empty. Also, `devbox shell` works fine. No errors.

`devbox generate direnv --force` made a lot of changes to my .envrc and fixed the issue. I can now use direnv in this directory without errors. Is this expected behavior?

**Comment by mohsenari** [comment]
@altano thanks for reporting this issue. We confirmed that generating a new .envrc file (`devbox generate direnv --force`) will fix this issue. This was due to a change that was backwards incompatible with .envrc files.
We are making a release today that fixes the backwards incompatibility. That said, generating a new .envrc like you did is the better solution to this issue.

**Comment by Lagoja** [comment]
This error should be resolved in 0.4.8, which added backwards compatibility for projects that set up the direnv integration prior to 0.4.7.
